src/database/schema/ChannelListSchema.ts

```diff
@@ -256,10 +267,16 @@ class ChannelList implements BaseDbSchema {
         B.last_active_at,
         B.profile_picture,
         B.bio,
-        B.is_banned
+        B.is_banned,
+        C.message,
+        C.type,
+        C.attachment_json as chat_attachment_json
       FROM ${ChannelList.getTableName()} A
       LEFT JOIN ${UserSchema.getTableName()} B
       ON A.last_updated_by = B.user_id AND A.id = B.channel_id
+      LEFT JOIN 
+        (SELECT DISTINCT(chats.channel_id), chats.* FROM chats ORDER BY chats.updated_at DESC LIMIT 1) as C
+      ON A.id = C.channel_id
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SQL query has been modified to include the `message`, `type`, and `attachment_json` fields from the `chats` table. However, there seems to be a potential issue with the subquery used to join the `chats` table. The `LIMIT 1` clause in the subquery might limit the result set to only one record, which may not be the intended behavior. If you want to get the latest chat for each channel, consider using a window function or a self-join instead.

```diff
- (SELECT DISTINCT(chats.channel_id), chats.* FROM chats ORDER BY chats.updated_at DESC LIMIT 1) as C
+ (SELECT chats.* FROM chats WHERE chats.updated_at IN (SELECT MAX(updated_at) FROM chats GROUP BY channel_id)) as C
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->